packages/wallets/solflare/src/metamask/wallet.ts

```diff
@@ -92,18 +85,11 @@ export class SolflareMetaMaskWallet implements Wallet {
                 version: '1.0.0',
                 signMessage: this.#signMessage,
             },
-            [SolflareMetaMaskNamespace]: {
-                solflareMetaMask: this.#instance,
-            },
```

Comment:
This has been removed to not expose the interface. You can expose this later if you want.